src/sqlfe/util/Utilities.java

```diff
@@ -226,4 +228,32 @@ public static String processSlashes(String path) {
 		return resultPath;
 	}	// end - method processSlashes
 	
+	public static boolean isQuestionFound(String line){
```

Comment:
Can we add a short single-line comment here and on Line 238?

Reply:
Added a single line comment for both the method definitions